File: pro2_9_27/kadai02e.c
//課題2e 2019-9-27 19d8104026j 可児憲太郎
#include<stdio.h>
#include<math.h>
int sosu(int input){
  int tmp=1;
  int i;
  if(input==1)return 0;
  if(input==2)return 1;
  for(i=2;i<=(int)sqrt(input)+1;i++){
    if(input%i==0){
      tmp=0;
      break;
    }
  }
  return tmp;
}
int main(void){
  int i;
  int sum;
  int year=0;
  for(i=1989;i<=2019;i++){
    sum=0;
    sum+=i%10+(i/10)%10+(i/100)%10+i/1000;
    if(sosu(sum)){
      printf("年:%d\n",i);
      printf("各桁の和:%d\n",sum);
      year++;
    }
  }
  printf("素数となった年の個数:%d\n",year);
  return 0;
}
/*

年:1990
各桁の和:19
年:1994
各桁の和:23
年:2000
各桁の和:2
年:2001
各桁の和:3
年:2003
各桁の和:5
年:2005
各桁の和:7
年:2009
各桁の和:11
年:2010
各桁の和:3
年:2012
各桁の和:5
年:2014
各桁の和:7
年:2018
各桁の和:11
素数となった年の個数:11
*/

File: kadai02e.py
#kadai02e 19d8104026j kaji kentaro 2000/9/28
def get_month(x):
	if x % 100 == 0 and x % 400 != 0:
		return 28
	elif x % 4 == 0:
		return 29
	else:
		return 28
days = [31,-1,31,30,31,30,31,31,30,31,30,31]
sy = int(input())
sm = int(input())-1
sd = int(input())-1
ey = int(input())
em = int(input())-1
ed = int(input())-1
sum = 0
for j in range(sy,ey+1):
	#last loop
	if j== ey:
		for i in range(sm,em+1):
			tmp = days[i]
			if tmp == -1:
				tmp = get_month(j)
			if i == em:
				tmp = ed
			#print(sum," ",tmp," ",sd)
			sum += tmp-sd
			sd = 0
			sm = 0
	#default loop
	else:
		for i in range(sm,12):
			tmp = days[i]
			if tmp == -1:
				tmp = get_month(j)
			#print(sum," ",tmp," ",sd)
			sum += tmp-sd
			sm = 0
			sd = 0
print(sum)
